sa/sa_test.go

```diff
@@ -4853,9 +4819,11 @@ func TestGetPausedIdentifiersOnlyUnpausesOneAccount(t *testing.T) {
 	sa, _, cleanUp := initSA(t)
 	defer cleanUp()
 
-	ptrTime := func(t time.Time) *time.Time {
-		return &t
-	}
+	defer func() {
+		// Drop all rows from the paused table.
+		_, err := sa.dbMap.ExecContext(ctx, "TRUNCATE TABLE paused")
```

Comment:
Why is `TRUNCATE TABLE` needed in this and other test functions? Isn't that covered by `ResetBoulderTestDatabase()`, called from `initSA()`'s returned `cleanUp` func?

Reply:
This above one is actually unnecessary. However, the rest are because calling `cleanUp()` closes the database before the next test case can run.

https://github.com/letsencrypt/boulder/blob/a0629f7fbe37172935ad22857cf2696e1653e31c/test/db.go#L57

Reply:
Ah, that makes sense, thanks! I hadn't noticed they were in table-driven tests.